GitCommands/Git/AheadBehindData.cs

```diff
@@ -2,15 +2,21 @@
 {
     public struct AheadBehindData
     {
+        public static readonly string Gone = "gone";
         public string Branch { get; set; }
         public string AheadCount { get; set; }
         public string BehindCount { get; set; }
 
         public string ToDisplay()
         {
-            return (!string.IsNullOrEmpty(AheadCount) ? AheadCount + "↑" : string.Empty)
-                   + (!string.IsNullOrEmpty(AheadCount) && !string.IsNullOrEmpty(BehindCount) ? " " : string.Empty)
-                   + (!string.IsNullOrEmpty(BehindCount) ? BehindCount + "↓" : string.Empty);
+            return AheadCount == Gone
+                ? Gone
+                : AheadCount == "0" && string.IsNullOrEmpty(BehindCount)
+                ? "0↑↓"
+                : (!string.IsNullOrEmpty(AheadCount) && AheadCount != "0"
+                    ? AheadCount + "↑" + (!string.IsNullOrEmpty(BehindCount) ? " " : string.Empty)
+                    : string.Empty)
+                    + (!string.IsNullOrEmpty(BehindCount) ? BehindCount + "↓" : string.Empty);
```

Comment:
Replace with a local function using StringBuilder or
```suggestion
                : (!string.IsNullOrEmpty(AheadCount) && AheadCount != "0"
                   ? AheadCount + "↑" + (!string.IsNullOrEmpty(BehindCount) ? " " : string.Empty)
                   : string.Empty)
                  + (!string.IsNullOrEmpty(BehindCount) ? BehindCount + "↓" : string.Empty);
```

Reply:
Changing the indentation